src/components/Background/index.css

```diff
@@ -17,10 +23,23 @@
   width: 100%;
   height: 100%;
 
-  background-color: color($light-blue alpha(28%));
-  mix-blend-mode: difference;
-
   @supports not (mix-blend-mode: difference) {
```

Comment:
All of these support calls will need to be on the `blend` classes, so each effect has a fallback.

Reply:
Added!